docs/developers/tutorial.md

```diff
@@ -562,12 +579,11 @@ class NoLiteralArguments(config: NoLiteralArgumentsConfig)
       .map(newConfig => new NoLiteralArguments(newConfig))
   }
   override def fix(implicit doc: SyntacticDocument): Patch = {
-    doc.tree
-      .collect {
+    doc.tree.collect {
         case Term.Apply(_, args) =>
           args.collect {
             case t: Lit if config.isDisabled(t) =>
-              Patch.lint(LiteralArgument(t))
+              Patch.lint(LiteralArgument(t)).atomic
```

Comment:
This was not change in [your PR](https://github.com/scalacenter/scalafix-named-literal-arguments/pull/9). That's because [lint patches are atomic by default](https://github.com/scalacenter/scalafix/blob/7462d5151ab7ed4a2d5b26da978a5d4efd3f73e8/scalafix-core/src/main/scala/scalafix/internal/patch/EscapeHatch.scala#L94-L98). 
```suggestion
              Patch.lint(LiteralArgument(t))
```